complexradio/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+      "name": "Tyler",
+      "id": "351991098387988491"
+    },
+    "service": "Complex Radio",
+    "description": {
+      "en": "Complex Radio provides quality music to its listeners, we also always have applications open for new presenters!"
+    },
+    "url": "complexr.pw",
+    "version": "0.1",
+    "logo": "https://i.imgur.com/FZvLxi4.png",
+    "thumbnail": "https://i.imgur.com/FZvLxi4.png",

```

Comment:
upload better quality logo
Thumbnail must be an actual thumbnail or a screenshot of the website